product-service/src/handlers/addProduct.js

```diff
@@ -0,0 +1,14 @@
+import { addProductToDb } from '../services/addProductToDb/addProductToDb';
+import { buildHttpResponse } from '../utils/buildHttpResponse';
+
+export const addProduct = async (event, context) => {
+    const { title = '', description = '', price = 0, image = '' } = JSON.parse(event.body);
+    console.log(JSON.parse(event.body))
```

Comment:
If the body is not passed you'll run into an unhandled error. And consequently, event data won't be logged